packages/config/src/configDefaults.ts

```diff
@@ -43,6 +43,10 @@ export const getInitialConfig = ({
       enabled: false,
       registryAddress: null
     },
+    mocha: {
```

Comment:
Currently we are relying on the fact that arbitrary flags that get passed get tacked on to the config object. I decided to make it official and create defaults for mocha.